### PR TITLE
Set up Electron integration and builds

### PR DESCRIPTION
**Pull request by aviraldg**

### CODE REVIEW

File: src/vector/index.js [hunk 1]
```diff
@@ -49,6 +48,7 @@ import UAParser from 'ua-parser-js';
 import url from 'url';
 
 import {parseQs, parseQsFromFragment} from './url_utils';
+import IntegrationManager from './integration';
```
Comment (dbkr) [inline src/vector/index.js]: We should probably call these something other than 'IntergrationManager' as that's going to confuse people with scalar. How about `PlatformManager` (or just `Platform`)?

**Comment by aviraldg** [comment]
OK, makes sense.

**Comment by dbkr** [comment]
Also, we should make sure we set up autoupdate before we publish an electron build (the first feature you write should always be auto-update...)

**Comment by dbkr** [comment]
Merging here: https://github.com/vector-im/vector-web/tree/aviraldg-electron

**Pull request closed without merging**